cmd/fts/cmd/run.go

```diff
@@ -0,0 +1,170 @@
+package cmd
+
+import (
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/gosuri/uiprogress"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var (
+	dryRun         bool
+	mainnet        bool
+	maxStagedBytes int64
+	maxDealBytes   int64
+	minDealBytes   int64
+	hiddenFiles    bool
+	taskFolder     string
+	resultsOut     string
+	concurrent     int64
+	ipfsrevproxy   string
+	retryErrors    bool
+)
+
+func init() {
+	runCmd.Flags().BoolVarP(&dryRun, "dry-run", "d", false, "Run through steps without pushing data to Powergate API.")
+	runCmd.Flags().BoolVarP(&mainnet, "mainnet", "n", false, "Sets staging limits based on localnet or mainnet.")
```

Comment:
`mainnet` flag is used below to toggle between two reasonable (?) sets of defaults for staging usage